### ISSUE TITLE
[BUG] Webpack + Bokehjs - Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

### ISSUE THREAD

**Issue by ashvin777** [bokeh/bokeh#9594]
I am using Webpack 4.x and BokehJS 1.4 to create different charts. While building the code, I am getting below error messages about Critical Dependencies.  However, the chart is rendered properly.  Is it possible to disable these warning statements??

```
//import
const Bokeh = require('bokehjs');

//package.json
"bokehjs": "1.4.0"
"webpack": "4.41.0",
"webpack-dev-server": "3.2.1",
```
```
Compiled with warnings.

./node_modules/bokehjs/build/js/lib/models/textures/canvas_texture.js
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

./node_modules/bokehjs/build/js/lib/models/transforms/customjs_transform.js
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

./node_modules/bokehjs/build/js/lib/models/transforms/customjs_transform.js
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

./node_modules/bokehjs/build/js/lib/models/callbacks/customjs.js
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

./node_modules/bokehjs/build/js/lib/models/formatters/func_tick_formatter.js
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

./node_modules/bokehjs/build/js/lib/models/filters/customjs_filter.js
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

./node_modules/bokehjs/build/js/lib/models/tools/inspectors/customjs_hover.js
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

```

**Comment by bryevdv** [comment]
@ashvin777 these warnings are not coming *from* Bokeh, and I am  not sure they can be turned off, that's really a webpack question. The models you list above are basically all "CustomJS" type models which are explicitly provided as a mechanism for dynamically injecting JS code from Python. Not having static dependencies is fundamentally part of what they are for. I'm not sure what we could do beyond bundle the CustomJS models separately from the main BokehJS, but that would be future work in a future release. 

cc @mattpap (As an aside, a separate bundle for CustomJS is actually possibly appealing, to help with #9411)

**Comment by mattpap** [comment]
This has been fixed in PR #9348 (2.0.0-dev). We don't use `require()` anymore. bokehjs' components should bundle without any warnings or errors in webpack and alike.

**Comment by mattpap** [comment]
> a separate CDN bundle for CustomJS is actually possibly appealing (...)

This seems too cumbersome. I think the solution suggested in #9411 is a better approach.
